casadm/cas_lib_utils.h

```diff
@@ -28,8 +28,13 @@ struct progress_status {
 void init_progress_bar(struct progress_status *ps);
 void print_progress_bar_or_indicator(float prog, struct progress_status *ps);
 int run_ioctl(int fd, int command, void *cmd);
+int run_ioctl_retry(int fd, int command, void *cmd);
 int run_ioctl_interruptible(int fd, int command, void *cmd,
 		char *friendly_name, int cache_id, int core_id);
+int run_ioctl_interruptible_retry(int fd, int command, void *cmd,
+		char *friendly_name, int cache_id, int core_id);
+int run_ioctl_interruptible_retry_option(int fd, int command, void *cmd,
+		char *friendly_name, int cache_id, int core_id, bool retry);
```

Comment:
These should be static and removed from header. They are helper functions and they should be accessible only locally.